plot-api/src/commonMain/kotlin/jetbrains/letsPlot/GlobalSettings.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022. JetBrains s.r.o.
+ * Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+ */
+
+package jetbrains.letsPlot
+
+import jetbrains.datalore.base.json.JsonSupport
+import jetbrains.datalore.plot.config.Option
+import jetbrains.letsPlot.intern.OptionsMap
+import jetbrains.letsPlot.intern.filterNonNullValues
+
+
+// Environment variables
+const val ENV_HTML_ISOLATED_FRAME = "LETS_PLOT_HTML_ISOLATED_FRAME"
+
+
+internal object GlobalSettings {
+    val isolatedFrameContext: Boolean get() = Env.getBool(ENV_HTML_ISOLATED_FRAME) ?: false
+    private val settings: MutableMap<String, String> = mutableMapOf()
+
+    fun set(pair: Pair<String, String>) {
+        settings += pair
+    }
+
+
+    private const val PLOT_THEME = "plot_theme"
+
+    fun setTheme(theme: OptionsMap?) {
```

Comment:
Let's handle theme similar to `isolatedFrameContext`:
- property (no get/set methods) holding an OptionsMap? value.
- read LETS_PLOT_PLOT_THEME env var to initialize.

Reply:
Done